oda_api/plot_tools.py

```diff
@@ -36,6 +36,9 @@
     pass 
 
 import logging
+logging.warning('please import oda_api.tools.plot instead of oda_api.plot_tools')
+
+from .tools.plot import *
```

Comment:
So, will this module disappear in order to be replaced by the one in `oda_api.tools?`?